examples/105-dotnet-CustomOptions/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.SemanticMemory;
+using Microsoft.SemanticMemory.Handlers;
+
+var memory = new MemoryClientBuilder()
+    .WithOpenAIDefaults(Env.Var("OPENAI_API_KEY"))
+    .WithOption(new TextPartitioningOption()
+    {
+        MaxTokensPerLine = 100,
+        MaxTokensPerParagraph = 300,
+        OverlappingTokens = 50,
+    })
+    .BuildServerlessClient();
+
+await memory.ImportDocumentAsync(new Document()
+    .AddFile("mswordfile.docx"), steps: new string[]
```

Comment:
Please use a smaller docx file.  We don't need a Word document with pictures for this test. If we keep this file, we'll increase the size of our repo by more than half a meg - even if we end up deleting this file in the future!

Reply:
this is actually a copy of a file already in the repo which we forgot to compress :-) I'll take care of it